src/db.rs

```diff
@@ -147,7 +164,11 @@ impl Db {
         let handle = self.keyspaces.read();
         let mut keyspaces = Vec::with_capacity(handle.keys().count());
         for key in handle.keys() {
-            keyspaces.push(Frame::String(key.clone()))
```

Comment:
Our protocol supports maps, so we can send the response as list of maps, where each keyspace will have its config in a map.

Maps in our protocol are built using vectors where each even index is key and each odd index is the value and the vector is even sized.

The response can be constructed like this.
```rust
let mut keyspaces = Vec::new();
let mut map = Vec::new();
map.push(Frame::String(Bytes::from("name"))); // this is the key
map.push(Frame::String(Bytes::from("<keyspace_name>"))); // this is the value
map.push(Frame::String(Bytes::from("evictor"))); // this is the key
map.push(Frame::String(Bytes::from_static(EVICTOR_RANDOM))); // this is the value
keyspaces.push(Frame::Map(map));
```

Reply:
I'm not sure that using a map as the response type makes sense as it is by nature unordered. We need to do the sorting on the utility side. Otherwise items won't come out in any guaranteed order, which leads to sometimes having weird sorts.

I think working with a `Vec` is probably the better way to do things 🤔